packages/node_modules/@ciscospark/sparkd/src/index.js

```diff
@@ -0,0 +1,66 @@
+/**!
```

Comment:
Please change these banners to 

```js
/*!
 *  Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
 */
```

The format has changed since the original PR opened.